.github/workflows/deploy.yaml

```
@@ -23,14 +24,16 @@ env:
   NODE_ENV: ${{ inputs.deploy-env }}
+  STATIC_DEPLOY_URL_PROD: mozhlyvosti.org.ua
+  STATIC_DEPLOY_URL_STAGING: staging-razom.vercel.app
 
 jobs:
   deployment:
     runs-on: ubuntu-latest
     timeout-minutes: 10
     environment:
       name: ${{ inputs.deploy-env }}
-      url: ${{ inputs.deploy-env == 'preview' && steps.deploy.outputs.url || 'https://razom.vercel.app/' }}
+       url: ${{ inputs.deploy-env == 'preview' && steps.deploy.outputs.url || (inputs.deploy-env == 'staging' && env.STATIC_DEPLOY_URL_STAGING || env.STATIC_DEPLOY_URL_PROD) }}
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 36: mapping values are not allowed in this context

(syntax-check)